nodeup/pkg/model/cloudconfig.go

```diff
@@ -74,6 +74,13 @@ func (b *CloudConfigBuilder) Build(c *fi.ModelBuilderContext) error {
 	}
 
 	config := "[global]\n" + strings.Join(lines, "\n") + "\n"
+
+	// We need this to support Kubernetes vSphere CloudProvider < v1.5.3
+	if fi.CloudProviderID(b.Cluster.Spec.CloudProvider) == fi.CloudProviderVSphere {
+		config += "[disk]\n" + "scsicontrollertype = pvscsi" + "\n"
```

Comment:
This code adds scsicontroller type in cloud config file of Kubernetes cloud provider. This will be used by Kubernetes vSphere Cloud Provider to attach the disks.